### PR TITLE
Focus input when chat message area clicked

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
Another option would be to do what the list scroll does and use state. This would eliminate the need for the external ref

**Comment by chromakode** [comment]
I think that using a ref is the best choice here since it's all in the same component hierarchy

File: shared/chat/conversation/index.desktop.js [hunk 1]
```diff
@@ -39,6 +41,7 @@ const Conversation = (props: Props) => {
         onEditMessage={props.onEditMessage}
         onLoadAttachment={props.onLoadAttachment}
         onLoadMoreMessages={props.onLoadMoreMessages}
+        onFocusInput={() => _input && _input.focusInput()}
```
Comment (chrisnojima) [inline shared/chat/conversation/index.desktop.js]: ok but can we make this a method and not an inline. let's try and avoid this pattern moving forward if possible

**Comment by chromakode** [comment]
@chrisnojima bump

**Pull request merged**